services/api-server/src/simcore_service_api_server/services/log_streaming.py

```diff
@@ -52,7 +54,18 @@ async def __aexit__(self, exc_type, exc, tb):
         await self.teardown()
 
     async def _distribute_logs(self, data: bytes):
```

Comment:
FYI: there is `servicelib.logging_utils.log_catch` decorator that does something very similar

Reply:
Yes, I know of that one. I will keep it like this for now. I guess we should remove this one after our experiments anyway.